assets/src/animation/utils/getTotalDuration.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function getTotalDuration({ animations = [] }) {
+  return animations.reduce(
+    (total, { duration = 0, delay = 0 }) => Math.max(total, duration + delay),
```

Comment:
Could you imagine an instance where these values are not numbers?

Reply:
Nope. But I can add some checks if you'd like. This is where TS would be nice to ensure that